packages/model-instance-handler/src/assertions.ts

```diff
@@ -145,29 +143,29 @@ export function assertValidUniqueValue(
  * if this check were to fail as part of a StreamtypeHandler's applyCommit function, that would
  * indicate a programming error.
  */
-export function assertEventLinksToState(
-  payload: DocumentDataEventPayload | TimeEvent,
-  state: DocumentState,
-) {
-  if (state.log.length === 0) {
-    throw new Error('Invalid document state: log is empty')
-  }
+// export function assertEventLinksToState(
+//   payload: DocumentDataEventPayload | TimeEvent,
+//   state: DocumentState,
+// ) {
+//   if (state.log.length === 0) {
+//     throw new Error('Invalid document state: log is empty')
+//   }
 
-  const initCID = state.log[0]
+//   const initCID = state.log[0]
 
-  // Older versions of the CAS created time events without an 'id' field, so only check
-  // the event payload 'id' field if it is present.
-  if (payload.id != null && payload.id.toString() !== initCID) {
-    throw new Error(
-      `Invalid init CID in event payload for document, expected ${initCID} but got ${payload.id}`,
-    )
-  }
+//   // Older versions of the CAS created time events without an 'id' field, so only check
+//   // the event payload 'id' field if it is present.
+//   if (payload.id != null && payload.id.toString() !== initCID) {
+//     throw new Error(
+//       `Invalid init CID in event payload for document, expected ${initCID} but got ${payload.id}`,
+//     )
+//   }
 
-  const prev = payload.prev.toString()
-  const expectedPrev = state.log[state.log.length - 1]
-  if (prev !== expectedPrev) {
-    throw new Error(
-      `Commit doesn't properly point to previous event payload in log for document ${initCID}. Expected ${expectedPrev}, found 'prev' ${prev}`,
-    )
-  }
-}
+//   const prev = payload.prev.toString()
+//   const expectedPrev = state.log[state.log.length - 1]
+//   if (prev !== expectedPrev) {
+//     throw new Error(
+//       `Commit doesn't properly point to previous event payload in log for document ${initCID}. Expected ${expectedPrev}, found 'prev' ${prev}`,
+//     )
+//   }
+// }
```

Comment:
can we delete this code if its not needed anymore? We can always get it back from git it we need it later.

Reply:
Done!